pkg/app/server/service/pipedservice/service.proto

```diff
@@ -146,7 +146,10 @@ service PipedService {
     // ListEvents returns a list of Events inside the given range.
     rpc ListEvents(ListEventsRequest) returns (ListEventsResponse) {}
     // ReportEventHandled marks the given all events as handled.
+    // Deprecated. This is only for the old Piped agents.
```

Comment:
Previously, we just marked it as handled, but that is not enough for the event list, that's why I settled on adding a new rpc.
Older versions of Piped can still handle events as before, and only `SUCCESS` and `NOT_HANDLED` can be seen in the UI. I'm going to notify users that they need to bring Piped up to date if they want to check the failure status.